src/shared/browser/types.ts

```diff
@@ -0,0 +1,19 @@
+export interface IBrowser {
+  storage: {
+    local: {
+      get: <Data>(key: string, defaultValue: Data) => Promise<Data>
+
+      set: <Data>(key: string, value: Data) => Promise<void>
```

Comment:
we can add more helpful logic here - setter should return `Promise<boolean>`, or even better - `Promise<boolean, Error>`